test/swingsetTests/escrow/bootstrap.js

```diff
@@ -163,11 +162,10 @@ expected successful check ${result}`;
     async bootstrap(argv, vats) {
       const host = await E(vats.host).makeHost();
       const randMintP = E(vats.mint).makeMint('rand');
-      const artMintP = makeMint(
-        'art',
-        makeBasicMintController,
-        makeUniAssayMaker(),
-      );
+
+      const { makeInviteConfig } = makeInviteConfigMaker();
```

Comment:
This shouldn't be an inviteConfigMaker. Art is an asset class, and it's not the same as invitations. I think invitations are a different kind of UniAsset, though it may be that there's no behavioral difference from assets where each asset is a particular named work. If the behavior is the same, I'd prefer it be named for the Uni assets rather than for the invites.